definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -773,6 +773,19 @@
     "RetriesTimeoutURLSubmittedEvent": "120,120",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "05/08/2019",
```

Comment:
why this date?

Reply:
As its ignored by CCD just copied and pasted from event above, will set to current date for clarity

Reply:
Addressed in 4b25a21f5bf186026c7933e86f859a8449132a4f